### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by rmarinho**
### Description of Change ###

Update readme 

### PR Checklist ###

- [ ] Has tests (if omitted, state reason in description)
- [ ] Rebased on top of master at time of PR
- [ ] Changes adhere to coding standard
- [ ] Consolidate commits as makes sense

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,23 +7,38 @@ Xamarin.Forms provides a way to quickly build native apps for iOS, Android, Wind
...
+
+![Visual Studio Team Services ](https://img.shields.io/vso/build/devdiv/0bdbc590-a062-4c3f-b0f6-9383f67865ee/7981.svg?style=flat&label=VSTS%20Master "VSTS")
+ 
 
-![OSX Debug Teamcity](https://img.shields.io/teamcity/https/teamcity.xamarin.com/e/XamarinForms_Debug_Cycle8ezTest_OsxDebug.svg?style=flat&label=OSX%20Debug%20%20%20%20%20 "OSX Debug")  
+## Packages ##
 
-![Windows Debug Teamcity](https://img.shields.io/teamcity/https/teamcity.xamarin.com/e/XamarinForms_Debug_Cycle8ezTest_WindowsDebug.svg?style=flat&label=Win%20Debug%20%20%20%20%20%20 "Win Debug")  
+##### Official Nuget.org ##### 
+![Xamarin Forms stable package](https://img.shields.io/nuget/v/Xamarin.Forms.svg?style=flat&label=stable "nuget")
```
Comment (hartez) [inline README.md]: Can we make these images into links so clicking on them takes you to nuget?

File: README.md [hunk 2]
```diff
@@ -53,6 +69,24 @@ Due to the way that Android works, the maps API key cannot be injected at runtim
...
+##### Build from the Command line #####
+Make sure you have Nuget.exe 4.0 or above and the a latest dotnet core sdk (2.0) 
+  
+
+     nuget restore Xamarin.Forms.sln
+     dotnet build Xamarin.Forms.sln
+
+
+##### API documentation changes #####
+If you change or add a public API on Xamarin.Forms the associated documentation should be updated and pushed as part of the pull request. This will also be needed to create a local nuget if you want. We rely on mdoc that is shipped on the repo on the tools folder. After building the .sln on Windows run:
```
Comment (hartez) [inline README.md]: "on the tools folder" -> "in the tools folder"

File: README.md [hunk 3]
```diff
@@ -53,6 +69,24 @@ Due to the way that Android works, the maps API key cannot be injected at runtim
 
 You can find out how to obtain a Google Maps API key [here](https://developer.xamarin.com/guides/android/platform_features/maps_and_location/maps/obtaining_a_google_maps_api_key/).
 
+##### Build from the Command line #####
+Make sure you have Nuget.exe 4.0 or above and the a latest dotnet core sdk (2.0)
```
Comment (samhouts) [inline README.md]: the ~a~ latest

File: README.md [hunk 4]
```diff
@@ -6,23 +7,38 @@ Xamarin.Forms provides a way to quickly build native apps for iOS, Android, Wind
...
+          <add key="NuGet.org" value="https://api.nuget.org/v3/index.json" />
+      </packageSources>
+  </configuration>
+  ```
 
-![Visual Studio Team Services Windows Debug](https://img.shields.io/vso/build/devdiv/0bdbc590-a062-4c3f-b0f6-9383f67865ee/6713.svg?style=flat&label=VSTS%20Win%20dbg "Win Debug VSTS")
+  *NOTE: This NuGet.Config should be with your application unless you want nightly packages to potentially start being restored for other apps on the machine.*
 
-![Visual Studio Team Services OSX Debug](https://img.shields.io/vso/build/devdiv/0bdbc590-a062-4c3f-b0f6-9383f67865ee/5514.svg?style=flat&label=VSTS%20OSX%20dbg "OSX Debug VSTS") 
+- Change your applications dependencies to have a `*` to get the latest version.
```
Comment (samhouts) [inline README.md]: application's

**Pull request merged**